components/plugins/lib.rs

```diff
@@ -70,5 +70,4 @@ fn register_clippy(reg: &mut Registry) {
 }
 #[cfg(not(feature = "clippy"))]
 fn register_clippy(reg: &mut Registry) {
```

Comment:
Should this function be removed altogether? After your changes, it does nothing.